airflow/providers/amazon/aws/transfers/redshift_to_s3.py

```diff
@@ -65,7 +65,7 @@ class RedshiftToS3Operator(BaseOperator):
     :type table_as_file_name: bool
     """
 
-    template_fields = ()
+    template_fields = (s3_key, )
```

Comment:
You need to pass `s3_key` as a string. I think that is the reason for the failing tests.

Reply:
thank you!